apps/prairielearn/src/pages/partials/questionNavSideButton.ejs

```diff
@@ -1,11 +1,13 @@
 <%
 let classes = ['btn', 'text-white', 'mb-3'];
-if (question.id && !question.sequence_locked) {
+let show_lock = false;
+if (question.id && !question.sequence_locked && (locals.group_role_permissions?.can_view ?? true)) {
```

Comment:
Nit, in my mind this would be easier to parse if we split it into the following conditions:

```ts
if (!question.id) {
  // IMO we should describe what this case is. Does this mean there's no question to link to?
  classes.push('btn-primary', 'disabled');
} else if (!question.sequence_locked && (locals.group_role_permissions?.can_view ?? true)) {
  classes.push('btn-primary');
} else {
  classes.push('btn-secondary', 'pl-sequence-locked');
  show_lock = true;
}
```

Reply:
Done in 2e83134.